sktime_dl/deeplearning/cnn/_regressor.py

```diff
@@ -90,7 +91,7 @@ def build_model(self, input_shape, **kwargs):
 
         model = keras.models.Model(inputs=input_layer, outputs=output_layer)
         model.compile(loss='mean_squared_error', optimizer=keras.optimizers.Adam(),
-                      metrics=['accuracy'])
+                      metrics=['mean_squared_error'])
```

Comment:
Well spotted, thanks.

Reply:
Apparently it still works with the accuracy though ... anyway, perhaps we should make the loss also a argument of the constructor?

Reply:
Nothing is being done with metrics here so it wasn't a problem.

Loss could be passed as an argument, yes, the user might like to try different loss functions.